doc/breaking-changes-0.7.md

```diff
@@ -0,0 +1,99 @@
+
+It includes the following new concepts:
+
+  * Role attributes
+     * Role attributes are just a set of strings associated to a model entity
+     * The semantics of the role attributes are determined by the system administrator 
+     * Ex: an edge router might have the role attributes `["us-east", "new-york", "omnicorp"]` 
+     * These tags might indicate that this edge router is located on the east coast of the USA, specifically in New York and should be dedicated to use by a customer named OmniCorp
+     * Currently role attributes are supported on edge routers and identities
+  * Roles 
+     * Roles specify a set of entities
+     * Roles may include role attributes as well as entity ids
+     * A role will match all entities which either:
+         * Have **_all_** role attributes in the role OR
+         * Have an ID which is listed explicitly
+     * Role attributes are prefixed with `@`. Role elements not prefixed with `@` are assumed to be ids
+     * There is a special role attribute `@all` which will match all entities
+     * A role may have only role attributes or only ids or may have both
+     
+## Role Example
+  * Edge router with id 1 has role attributes `["us-east", "New York City"]`    
+  * Edge router with id 2 has role attributes `["us-east", "Albany"]`    
+  * Edge router with id 3 has role attributes `["us-west", "Los Angeles"]`
+  * An edge router role of `["@us-east", "@New York City", "3"]` would evaluate as follows
+     * Edge router 1 would match because it has all listed role attributes
+     * Edge router 2 would not match, because it doesn't have all listed role attributes
+     * Edge router 3 would match because its ID is listed explicitly
+     
+## Model Changes
+### Role Attributes
+Edge routers and identities now have roleAttributes fields.
+
+### Edge Router Policies
+0.7.0 introduces a new model construct, the Edge Router Policy. This entity allows restricting which edge routers identities are allowed to use. An edge router policy has three attributes:
+
+  * Name
+  * Identity Roles
+  * Edge Router Roles
+  
+An identity can be a member of multiple policies and will have access to the union of all edge routers linked to from those policies.
+
+There is a new `/edge-router-policies` endpoint which can be used for creating/updating/deleting/querying edge router policies. Edge router policies PUT/POST/PATCH all take the following properties:
+
+  * name
+  * edgeRouterRoles
+  * identityRoles
+  * tags
+
+### Service Edge Router Roles
+Services now have a new edgeRouterRoles field. If set, this specifies which edge routers may be used for a service. This replaces the old cluster functionality. 
+
+### Edge Router Access 
+When a service is dialed or bound, which edge routers will be returned?
+
+  * If the service edgeRouterRoles are NOT set, then it will the set of edge routers that the dialing/binding identity has access to 
```

Comment:
`it will the set` -> `it will be the set`

Reply:
Fixed